calico-cloud/threat/security-event-management.mdx

```diff
@@ -83,12 +83,12 @@ No. {{prodname}} security events do not encompass all types of alerts nor all se
 
 **What does dismissing a security event do?**
 
-Dismissing a security event hides it from view. Use the toggle button to show/hide. There is no option to undo a dismissed alert.
+Dismissing a security event hides it from view. Use the toggle button to show/hide. If dismissed by mistake, a security event can be restored.
```

Comment:
If the board has been updated and both dismissing and restoring are intuitive, I'd get rid of the entire section. But if restoring is not intuitive, let's be more specific:

If you dismiss an event in error, you can restore it by...

Reply:
It's intuitive and uses the same workflow (if an item is dismissed, you can click "Restore Security Event"; if not dismissed you can click "Dismiss Security Event").

I think we could leave it as is to keep "Dismissing a security event hides it from view" as some people may be wondering "what does dismiss do?"

Reply:
Agree. The bullet should read:

- Dismissing a security event hides it from view